src/python/serenity_types/pricing/derivatives/options/volsurface.py

```diff
@@ -128,6 +154,11 @@ class VolPoint(BaseModel):
 
 
 class RawVolatilitySurface(BaseModel):
+    strike_type: StrikeType
+    """
+    Strike representation used for this surface, e.g. ABSOLUTE or LOG_MONEYNESS.
+    """
+
```

Comment:
Should we add strike_type here?
From code, we are generating different vol_surface rawbased on different strike_type